### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by Jineshdarjee**
## Description
* Added unit test cases for the Paymaster > index.ts file

### CODE REVIEW

**Review by IAmKio — APPROVED** [review]
LGTM

File: backend/src/paymaster/pimlico.test.ts [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+    try {
+      await pimlicoPaymaster.calculateTokenAmount(userOp);
+    } catch (e) {
+      throw new Error('An error is displayed while performing calculateTokenAmount action.')
+    }
+  });
+
+  test('generatePaymasterAndData function - valid details', async () => {
+    try {
+      await pimlicoPaymaster.generatePaymasterAndData(userOp);
```
Comment (ch4r10t33r) [inline backend/src/paymaster/pimlico.test.ts:47]: I don't see any assertion here? How is this supposed to work? What am I missing?

File: backend/src/paymaster/pimlico.test.ts [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+    const ERC20PaymasterBuildOptions = {
+      entrypoint: "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789",
+      nativeAsset: NATIVE_ASSET[80001],
+      nativeAssetOracle: ORACLE_ADDRESS[80001].MATIC,
+      tokenAddress: TOKEN_ADDRESS[80001].USDC,
+      tokenOracle: ORACLE_ADDRESS[80001].USDC,
+      deployer: "0x4337000c2828f5260d8921fd25829f606b9e8680"
+    }
+    try {
+      await getERC20Paymaster(provider, erc20, entryPointAddress, ERC20PaymasterBuildOptions);
```
Comment (ch4r10t33r) [inline backend/src/paymaster/pimlico.test.ts:82]: Please above. How is this a unit test? Where are we checking for happy path?

**Review by ch4r10t33r — CHANGES_REQUESTED** [review]

**Comment by Jineshdarjee** [comment]
## Description
* Added the assertion in the unit test cases 
* Bifurcated the smoke (happy path) and regression test cases for better clarity.

CC: @ch4r10t33r

**Review by ch4r10t33r — APPROVED** [review]

**Review by vignesha22 — CHANGES_REQUESTED** [review]
Please undo the deletion of config.json.default pls since it's the initial setup file required

**Comment by Jineshdarjee** [comment]
@vignesha22 Added config.json file.

**Review by vignesha22 — APPROVED** [review]
LGTM

**Pull request merged**